### DISCUSSION TITLE
Missing show all databases option in Azure SQL connection settings

### DISCUSSION THREAD

**Discussion by okruga** [dbeaver/dbeaver#35983] [General]
Hello
Is there an option to enable something like "Show all databases" for Azure SQL connection?
DBeaver version: 24.2.2.202410061621 Community edition
MacOS for Apple Silicon

<img width="715" alt="image" src="https://github.com/user-attachments/assets/62157518-b468-447a-93e2-bb57e03cf290">

**Comment by TB-HunBoka** [comment]
For Snowflake if you hit test connection, then it will query all database allowed for the user.
You can try that for Azure SQL.

**Reply by okruga** [reply]
thanks for the reply, but doesn't work for me.
It just connects to master if database name is not specified.

**Reply by TB-HunBoka** [reply]
> thanks for the reply, but doesn't work for me. It just connects to master if database name is not specified.

Probably you did the right thing, but based on your response, and re reading what I wrote, I think I could explain better what I wanted to say.

When you are editing the connection, could you click on the test connection button. Wait for the connection to be verified, then in the connection editor clicking in the Database/Schema field. Does it make a popup menu with available databases?

**Reply by okruga** [reply]
same behaviour. No popup.
And even if there was one, I want to select database after connection to the server, expecting to have the list of all of them under "databases" folder

**Comment by E1izabeth** [comment]
Issue about that #36162
We will return this option in the next release
